.github/workflows/all-packages.yml

```diff
@@ -156,6 +185,19 @@ jobs:
           toolchain: stable
```

Comment:
Are you sure that you aren't doing duplicate stuff here?

Reply:
There are more CI related commits in v0.24 so I would back port all together otherwise it will be a mess

Reply:
Agreed to keep it as it is since CI doesn't work in 25 anyways and then make a separate PR to put the workflows in v0.25 in order